packages/block-library/src/search/block.json

```diff
@@ -0,0 +1,24 @@
+{
+	"name": "core/search",
+	"title": "Search",
+	"category": "widgets",
+	"icon": "search",
+	"description": "Help visitors find your content.",
+	"keywords": [ "find" ],
+	"attributes": {
+		"label": {
+			"type": "string",
+			"default": "Search"
```

Comment:
`label` and `buttonText` are tricky, too. Their default values were originally wrapped with i18n function. How do we apply it through JSON file?